src/universal/modules/outcomeCard/containers/OutcomeCard/OutcomeCardContainer.js

```diff
@@ -72,6 +85,19 @@ class OutcomeCardContainer extends Component {
     });
   };
 
+  isEditing = () => {
+    return !this.state.activeEditingClients.isEmpty()
+  }
+
+  trackEditingClient = (uid, isEditing) => {
+    this.setState((curState) => {
```

Comment:
ah yeah, the fickle async nature of setState is a big pain!
A race condition still exists here though (although it's out of scope for this batting practice, you've already crushed it, so feel free to ignore!)

For example, let's say I wanted to expand this logic to the card's menus (surely if i'm changing the status of the project, I'm still "editing", right?).
When I mouseDown that status menu toggle, the focus will be taken away from the text area, leaving us with 0 clients, and thus triggering an announce that i'm not editing. then, once the menu pops up, it'll add a new editing client to the Set, thus triggering another announce in quick succession. How might I avoid this & just keep the "editing" status?